core/src/main/java/org/opensearch/sql/expression/ExpressionNodeVisitor.java

```diff
@@ -26,9 +26,10 @@ public T visitNode(Expression node, C context) {
 
   /**
    * Visit children nodes in function arguments.
-   * @param node      function node
-   * @param context   context
-   * @return          result
+   *
+   * @param node function node
+   * @param context context
+   * @return result
```

Comment:
another spacing issue

Reply:
I think it's an improvement removing the extra spaces